homework/express-cat/app.js

```diff
@@ -0,0 +1,51 @@
+var express = require('express');
+var path = require('path');
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+var exphbs  = require('express-handlebars');
+var mongoose = require('mongoose');
+
+
+var index = require('./routes/index');
+var catroutes = require('./routes/catroutes');
+var models = require('./modelsdb')
+
+var db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
```

Comment:
Could just as easily have done `db.on('error', console.error)`. Using `bind` is unnecessary.